browser/brave_browser_main_parts.cc

```diff
@@ -161,6 +162,10 @@ void BraveBrowserMainParts::PostBrowserStart() {
     }
   }
 #endif  // !BUILDFLAG(IS_ANDROID)
+
+#if !BUILDFLAG(IS_ANDROID)
+  ipfs::CleanupIpfsComponent();
```

Comment:
AFAIU moving the code from components to chrome was done only becase of `chrome::USER_DATA`. If that's correct, I'd suggest to move only this call (and pass the FilePath with user data dir into it) and keep the component cleaner in components, it just looks more logical to me.

Reply:
done